api/RTCStatsReport.json

```
@@ -255,9 +258,8606 @@
+                "version_added": "27"
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "15.3",
+                "version_removed": "15.4"
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "retransmittedBytesSent": {
+          "__compat": {
+            "description": "<code>retransmittedBytesSent</code> in 'outbound-rtp' stats",
+            "spec_url": "https://w3c.github.io/webrtc-stats/#dom-rtcoutboundrtpstreamstats-retransmittedbytessent",
+            "support": {
+              "chrome": {
+                "version_added": "≤80"
+              },
+              "chrome_android": {
+                "version_added": "≤112"
+              },
+              "edge": {
+                "version_added": "≤80"
+              },
+              "firefox": {
+                "version_added": "99> ≤101"
+              },
+              "firefox_android": {
+                "version_added": "≤112"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "13.1> ≤14.1"
+              },
+              "safari_ios": {
+                "version_added": "≤16.4"
+              },
+              "samsunginternet_android": {
+                "version_added": "≤20.0"
+              },
+              "webview_android": {
+                "version_added": "≤112"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "retransmittedPacketsSent": {
+          "__compat": {
+            "description": "<code>retransmittedPacketsSent</code> in 'outbound-rtp' stats",
+            "spec_url": "https://w3c.github.io/webrtc-stats/#dom-rtcoutboundrtpstreamstats-retransmittedpacketssent",
+            "support": {
+              "chrome": {
+                "version_added": "≤80"
+              },
+              "chrome_android": {
+                "version_added": "≤112"
+              },
+              "edge": {
+                "version_added": "≤80"
+              },
+              "firefox": {
+                "version_added": "99> ≤101"
```

https://mdn-bcd-collector.gooborg.com/tests/api/RTCStatsReport/type_outbound-rtp/retransmittedPacketsSent

Tests shown added in 100

```suggestion
                "version_added": "100"
```